### ISSUE TITLE
Is it possible to update the version of importlib-metadata?

### ISSUE THREAD

**Issue by Taikono-Himazin** [django-ses/django-ses#253]
I'm using django-ses and djlint.
I got the following warning when I installed django-ses.

```
djlint 1.9.3 requires importlib-metadata<5.0.0,>=4.11.0, but you have importlib-metadata 1.7.0 which is incompatible.

django-ses 3.1.0 requires importlib-metadata<2.0,>=1.0; python_version < "3.8", but you have importlib-metadata 4.12.0 which is incompatible.
```
I found that the import-metadata version specification that django-ses depends on is out of date. Is it possible to update this version?

**Comment by pcraciunoiu** [comment]
@Taikono-Himazin sure, it appears this is only used here: https://github.com/django-ses/django-ses/blob/63cc50932bef1960be4f10dc92566c8f1c2a86c7/django_ses/__init__.py#L12-L19

Would you be interested in opening a Pull Request to change the requirement? I am pretty sure you can update `pyproject.toml` to specify a wildcard instead of a specific version, or a `>=1` if that gets rid of it. Then you may just need to run `poetry update` in this repo and open a PR.

**Comment by pcraciunoiu** [comment]
The PR has been merged and released as 3.1.2 on pypi - https://pypi.org/project/django-ses/3.1.2/

**Comment by Taikono-Himazin** [comment]
I think this issue has been resolved, so I will close it.
Thank you.

I don't know much about the test failure issue, so I think it's difficult to raise a PR. Should I just delete the test case?

**Comment by pcraciunoiu** [comment]
No worries, I fixed in #256 by mocking properly.
